core/src/main/scala/akka/kafka/ConsumerMessage.scala

```diff
@@ -28,6 +28,15 @@ object ConsumerMessage {
       committableOffset: CommittableOffset
   )
 
+  /**
+   * Output element of `transactionalSource`.
+   * The offset is automatically committed as by the Producer
+   */
+  final case class TransactionalMessage[K, V](
+      record: ConsumerRecord[K, V],
+      partitionOffset: PartitionOffset
+  )
```

Comment:
Did you forget to make this class implement `PartitionOffsetMessage` ?

Reply:
This is an artifact of an earlier implementation attempt where I tried to generalize consumer group committable messages and transactional messages by extracting `PartitionOffsetMessage` into a trait.  I didn't pursue this option because I thought it would be more appropriate for the `ProducerStage` to control the transaction commits, like they're done in Kafka Streams.  For now I'll remove the `PartitionOffsetMessage` since it's not being used.